projects/js-packages/publicize-components/src/store/effects.js

```diff
@@ -19,7 +20,9 @@ import { SUPPORTED_CONTAINER_BLOCKS } from '../components/twitter';
  */
 export async function refreshConnectionTestResults() {
 	try {
-		const results = await apiFetch( { path: '/wpcom/v2/publicize/connection-test-results' } );
+		const connectionRefreshPath =
+			getJetpackData()?.connectionRefreshPath ?? '/wpcom/v2/publicize/connection-test-results';
```

Comment:
Similarly to the Jetpack Cloud connections screen, we don't want to default to the new endpoint for WPCOM until we've moved to the new package. This allows us to configure the URL used in the initial state.